webapp/components/Chat/Chat.vue

```diff
@@ -254,8 +255,19 @@ export default {
           fetchPolicy: 'no-cache',
         })
 
+        const newMsgIds = Message.filter((m) => m.seen === false).map((m) => m.id)
+        if (newMsgIds.length) {
+          this.$apollo.mutate({
+            mutation: markMessagesAsSeen(),
+            variables: {
+              messageIds: newMsgIds,
+            },
+          })
+        }
+
         const msgs = []
         ;[...this.messages, ...Message].forEach((m) => {
+          m.seen = true
```

Comment:
only those from the other user? You currently also mark your own messages as seen?!